sdk/schemaregistry/schema-registry-avro/package.json

```diff
@@ -74,6 +74,7 @@
     "@azure/schema-registry": "1.0.2",
     "avsc": "^5.5.1",
     "buffer": "^6.0.0",
+    "lru-cache": "^6.0.0",
```

Comment:
One small thing to look at is I know we have a list of "blessed" dependencies and a guideline to avoid other external dependencies. But I don't know how up-to-date this guideline is so up to you if you want to follow up on it https://azure.github.io/azure-sdk/typescript_implementation.html#ts-dependencies-no-other-packages

Reply:
This one's a pretty core JS ecosystem tool. It has 66M weekly downloads (4x more than React).

Reply:
I'm good with this one, it's already in our graph in several places